genesyscloud/tfexporter/resource_genesyscloud_tf_export_test.go

```diff
@@ -380,6 +393,12 @@ func TestAccResourceTfExportIncludeFilterResourcesByRegEx(t *testing.T) {
 			[]string{
 				strconv.Quote("genesyscloud_routing_queue::-prod"),
 			},
+			[]string{
```

Comment:
This was the cause of one of the bugs where I was getting a nil pointer exception.  The problem is that essentially we were creating the resources we were using in the test at the same time we were executing the export.  This is a race condition because the resources that the exporter test is executing might not be complete before the test cases complete.  

This is why sometimes we do not see the test failing.  So I added an explicit depends_on to each test case to ensure that resources that are needed by the exporter test case are created before the exporter starts.  This code creates a list of resources that needs to be passed into resource creation.